pdb2sql/pdb2sql_base.py

```diff
@@ -127,27 +130,95 @@ def exportpdb(self, fname, append=False, periodic=False, **kwargs):
             line = 'ATOM  '
             line += '{:>5}'.format(d[0])    # serial
             line += ' '
-            line += '{:^4}'.format(d[1])    # name
+            line += self._format_atomname(d) # name
             line += '{:>1}'.format(d[2])    # altLoc
-            line += '{:>3}'.format(d[3])  # resname
+            line += '{:>3}'.format(d[3])    # resname
             line += ' '
             line += '{:>1}'.format(d[4])    # chainID
             line += '{:>4}'.format(d[5])    # resSeq
             line += '{:>1}'.format(d[6])    # iCODE
             line += '   '
-            line += '{: 8.3f}'.format(d[7])  # x
-            line += '{: 8.3f}'.format(d[8])  # y
-            line += '{: 8.3f}'.format(d[9])  # z
+            line += pdb2sql_base._format_xyz(d[7]) # x
+            line += pdb2sql_base._format_xyz(d[8]) # y
+            line += pdb2sql_base._format_xyz(d[9])  # z
             if not self.no_extra:
-                line += '{: 6.2f}'.format(d[10])    # occ
-                line += '{: 6.2f}'.format(d[11])    # temp
+                line += '{:>6.2f}'.format(d[10])    # occ
+                line += '{:>6.2f}'.format(d[11])    # temp
+                line += ' ' * 10
+                line += '{:>2}'.format(d[12])       # element
             line += '\n'
 
             f.write(line)
 
         # close
         f.close()
 
+    def _format_atomname(self, data):
```

Comment:
That's nice and controlled but is it needed to read the data ? But at least to write it I think it's really useful

Reply:
I assume the input PDB has strict format, so we don't it that for reading. But we do need some code to preprocess PDB files, I'll write it later.